Harvest.Web/ClientApp/src/Expenses/ExpenseEntryContainer.tsx

```diff
@@ -131,6 +131,9 @@ export const ExpenseEntryContainer = () => {
     const newActivityId = Math.max(...activities.map((a) => a.id), 0) + 1;
     setActivities((acts) => [...acts, getDefaultActivity(newActivityId)]);
   };
+// return true if the sum of the acctivity totals is greater than 0
+  const isValid = (activities: Activity[]) =>
+    activities.reduce((prev, curr) => prev + curr.total || 0, 0) > 0;
```

Comment:
Why have an `activities` parameter? It would work just as well doing:
```
  const isValid = () =>
    activities.reduce((prev, curr) => prev + curr.total || 0, 0) > 0;
```